### PR TITLE
Homework of 2021-finalproject

### PR DESCRIPTION
**Pull request by TNTksals**
I have finished the homework which assigned by my teacher, so I want to submit my homework.

### CODE REVIEW

**Comment by TNTksals** [comment]
I think I need a reviewer. @MuZhou233

File: 2021-finalproject/TNTksals/scripts/check-environment.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```
Comment (MuZhou233) [inline 2021-finalproject/TNTksals/scripts/check-environment.sh:1]: This script will be better if print detail information after a test failed

**Review by MuZhou233 — APPROVED** [review]

File: 2021-finalproject/TNTksals/scripts/check-environment.sh (hunk 1 above)
Comment (TNTksals) [reply 2021-finalproject/TNTksals/scripts/check-environment.sh:1]: > This script will be better if print detail information after a test failed

I have added the information about the failed test. Anything else you would like me to do? In addition, I don't know why I dismissed your stale review. I am sorry. Could you reviewe me again?

Comment (MuZhou233) [reply 2021-finalproject/TNTksals/scripts/check-environment.sh:1]: > I have added the information about the failed test. Anything else you would like me to do? In addition, I don't know why I dismissed your stale review. I am sorry. Could you reviewe me again?

Don't worry, your new commit will dismiss review automatically

**Review by MuZhou233 — DISMISSED** [review]

**Comment by TNTksals** [comment]
I have finished to commit my homework to the renamed folder, please review me again.

**Review by MuZhou233 — APPROVED** [review]

**Pull request merged**